src/Mailer/MailSentCommandHandler.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailer;
+
+use Broadway\CommandHandling\CommandHandler;
+use CultuurNet\UDB3\CommandHandling\AsyncCommand;
+use CultuurNet\UDB3\CommandHandling\AsyncCommandTrait;
+use CultuurNet\UDB3\Mailer\Event\SentMail;
+use DateTimeImmutable;
+use Psr\Log\LoggerInterface;
+
+class MailSentCommandHandler implements CommandHandler, AsyncCommand
```

Comment:
Since the command is `SendMail` this should probably be `SendMailCommandHandler`

Reply:
Changed to `SendOwnershipMailCommandHandler`